### DISCUSSION TITLE
Need suggestion for : nxp mpc5748g flashing

### DISCUSSION THREAD

**Discussion by pavanrj2** [pyocd/pyOCD#1162] [Q&A]
Hi  !! 
I thank you for giving your time for this query , 
I couldn't find a linux supported tool like pyOCD that could be used  to flash "nxp mpc5748g"  board . 
could you please let me know if you know / used any in the past . 

I failed to find One .

Thanks again

**Comment by flit** [comment]
Sorry, I'm not aware of a similar tool for the NXP PPC devices. Not open source at least.

**Reply by pavanrj2** [reply]
Thank you for the reply !
